### DISCUSSION TITLE
naive question about GUI

### DISCUSSION THREAD

**Discussion by oczt** [libre-tube/LibreTube#511] [General]
Please do not take this question to be offensive - I apologise if you think so. It is just a honest curiosity.

1. Is there are reason to rewrite this GUI totally? Would it be not easier to fork NewPipe and change the backend suitably?
2. Or at least keep GUI very similar to NewPipe (instead of reinventing).

The main reason I ask is that there are so many GUI related improvements w.r.t. user convenience etc already polished (reasonably) by NewPipe.

Thanks

**Comment by Bnyro** [comment]
We wrote LibreTube from scratch, we don't use any code from NewPipe.

**Comment by Adithyan007hack** [comment]
But I think newpipe ui is a more outdated
Libretube ui looks clean and simple to me

**Comment by oczt** [comment]
Hmmm.. Dont you think there is so much wasted space in LibreTube? Especially the like-share-download-VLC-Save (icon bar - with tint/highlight) and so much space padding below/above it. (PS: I use a 5 inch smartphone).
